integration-cli/docker_api_exec_resize_test.go

```diff
@@ -27,56 +25,29 @@ func (s *DockerSuite) TestExecResizeAPIHeightWidthNoInt(c *check.C) {
 
 // Part of #14845
 func (s *DockerSuite) TestExecResizeImmediatelyAfterExecStart(c *check.C) {
+	testRequires(c, DaemonIsLinux)
```

Comment:
How did this test pass previously on Windows?

Reply:
There are some big problems with this test. I think it's actually failing on windows, but all ignores are being ignored because it's ignoring all errors except EOF: https://github.com/moby/moby/blob/733ed2ddd3c621dadafbb74feb7b80d20fd3fd6f/integration-cli/docker_api_exec_resize_test.go#L68-L79

I tried to fix it, but it was failing, so I decided to leave it for another time.

Reply:
Opened #36354 for this issue